lib/tasks/deployment/20230109153235_populate_states.rake

```diff
@@ -0,0 +1,21 @@
+require "csv"
+
+namespace :after_party do
+  desc "Deployment task: populate_states"
+  task populate_states: :environment do
+    puts "Running deploy task 'populate_states'"
+
+    filename = File.join(Rails.root, "lib", "tasks", "files", "state.txt")
```

Comment:
nit: we could rename the file to `state.csv`, since it seems to be a csv and not a txt

Reply:
Well, it is not comma-separated, and doesn't actually open nicely in Numbers, so I figured it was better to leave it as `.txt`. It seems that the CSV library is smart enough to handle it, but I don't think it's technically a CSV file?